config/locales/en.yml

```diff
@@ -143,6 +196,10 @@ en:
             active: Active
             pending: Pending
           status_label: Status
+  permissions:
```

Comment:
remove line 199-202 for this and all other languages

Reply:
4 line block starting with permissions: has been deleted.